_data/kernels.json

```diff
@@ -215,6 +215,14 @@
 
         [ "--enable", "CONFIG_INET_DIAG"],
         [ "--enable", "CONFIG_INET_UDP_DIAG"],
-        [ "--enable", "CONFIG_INET_DIAG_DESTROY"]
+        [ "--enable", "CONFIG_INET_DIAG_DESTROY"],
+
+        [ "--enable", "CONFIG_INTEGRITY"],
+        [ "--enable", "CONFIG_IMA"],
+        [ "--enable", "CONFIG_IMA_MEASURE_PCR_IDX"],
```

Comment:
@tpapagian fyi I'm seeing

> #33 5.247 time="2025-03-18T11:49:05Z" level=warning msg="stderr> .config:7811:warning: symbol value 'y' invalid for IMA_MEASURE_PCR_IDX\n"

which makes sense, as it's a range of values. Should we just leave it at the default value then?

Reply:
makes sense, so I assume that you have just to remove that?

Reply:
https://github.com/cilium/little-vm-helper-images/pull/899